pkg/kafka/utils.go

```diff
@@ -1,7 +1,8 @@
 package kafka
 
 var (
-	Clouds         = []string{"aws", "azure", "gcp"}
-	Availabilities = []string{"single-zone", "multi-zone"}
-	Types          = []string{"basic", "standard", "enterprise", "dedicated"}
+	Clouds            = []string{"aws", "azure", "gcp"}
+	Availabilities    = []string{"single-zone", "multi-zone"}
+	Types             = []string{"basic", "standard", "enterprise", "dedicated"}
+	BYOCSupportClouds = []string{"aws", "azure"}
```

Comment:
```suggestion
	ByocSupportedClouds = []string{"aws", "azure"}
```

Reply:
What does BYOC stand for? If "C" stands for connect, maybe this belongs in the "connect" package instead of the "kafka" package?